### PR TITLE
Remove pluggable delegates

### LINKED ISSUE #64: Remove pluggable application delegates
As we no more need pluggable app delegates, we should remove them also from template.

https://github.com/AckeeCZ/iOS-MVVM-ProjectTemplate/blob/08a08cbac9bcebdfc85f451750d43eb08e9a01f5/ProjectTemplate/AppDelegates/Base/PluggableApplicationDelegate.swift#L13

### PR DESCRIPTION
**Pull request by olejnjak**
Remove pluggable app delegates as we no more use them as most of logic it should be used for has moved to flow coordinators.

Closes #64.

#### Checklist

- [x] Updated CHANGELOG.md.

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -2,14 +2,14 @@ PODS:
...
   - Smartling.i18n (1.0.14)
-  - SwiftGen (6.2.0)
-  - SwiftLint (0.39.2)
+  - SwiftGen (6.3.0)
+  - SwiftLint (0.40.3)
 
 DEPENDENCIES:
   - ACKLocalization (~> 0.3)
   - LicensePlist (~> 2.14)
   - Smartling.i18n (~> 1.0)
```
Comment (LukasHromadnik) [inline Podfile.lock:11]: ❓ I think we can remove Smartling since SwiftGen is updated to 6.3. Should we do that in separate PR?

**Review by LukasHromadnik — COMMENTED** [review]
❓ What about FirebaseAppDelegate? Is it still in the project?

File: Podfile.lock (hunk 1 above)
Comment (olejnjak) [reply Podfile.lock:11]: This will be separate MR considering #56.

**Comment by olejnjak** [comment]
> ❓ What about FirebaseAppDelegate? Is it still in the project?

Removed ✅

**Review by LukasHromadnik — APPROVED** [review]

**Review by fortmarek — APPROVED** [review]
